provisioning/playbooks/couchbase/create_bucket.yml

```diff
@@ -0,0 +1,34 @@
+---
+- hosts: couchbase
+  sudo: False
+  gather_facts: False
+  tasks:
+    - name: Create cluster grouping
+      group_by: key={{ couchbase_server_node_role }}
+
+- hosts: primary
+  sudo: False
+  gather_facts: False
+  vars:
```

Comment:
The variables are definitely related to the server and not just for the bucket.
bucket_ram is related to server_ram.
bucket_name is related to default_bucket.